docs/index-backend/text-search.md

```diff
@@ -59,6 +59,7 @@ Full-text search is case-insensitive.
 -   `textContainsFuzzy`: is true if (at least) one word inside the text
     string is similar to the query String (based on Levenshtein edit
     distance)
+- `textContainsPhrase`:  is true if the text string does contain the sequence of words in the query string
```

Comment:
nitpick: same indentation as the above ones. If you don't mind, can you also add an example below (line 70) like the other predicates?

Reply:
Whoops 🤦‍♂️ . Yes of course will fix the indent and happy to add an example.